test/integration/daemonset/daemonset_test.go

```diff
@@ -278,11 +278,13 @@ func newNode(name string, label map[string]string) *v1.Node {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Labels:    label,
-			Namespace: metav1.NamespaceDefault,
+			Namespace: metav1.NamespaceNone,
 		},
 		Status: v1.NodeStatus{
 			Conditions:  []v1.NodeCondition{{Type: v1.NodeReady, Status: v1.ConditionTrue}},
 			Allocatable: v1.ResourceList{v1.ResourcePods: resource.MustParse("100")},
+			// minimum version required to use matchFields
+			NodeInfo: v1.NodeSystemInfo{KubeletVersion: "v1.11.0"},
```

Comment:
Do we have any tests for kubelet 1.10?

Reply:
nop, let me add one for this case. There is a discussion at #69346 to include 2 version upgrade check in e2e.

Reply:
done. mix the kubelet with 1.10 and 1.11 :)